### ISSUE TITLE
local invocation in README is broken problem 2: BAM loading?

### ISSUE THREAD

**Issue by timodonnell** [hammerlab/guacamole#511]
This is with the fix in #510 

Can't load local bams anymore?

```
[tim@Tims-MacBook ~/sinai/git/guacamole]$ scripts/guacamole somatic-joint     src/test/resources/synth1.normal.100k-200k.withmd.bam     src/test/resources/synth1.tumor.100k-200k.withmd.bam     --reference-fasta ~/sinai/data/b37.fasta     --out /tmp/out.vcf
Found guac jar: target/guacamole-0.0.1-SNAPSHOT.jar
Java HotSpot(TM) 64-Bit Server VM warning: ignoring option MaxPermSize=512m; support was removed in 8.0
--> [Tue Jul 26 14:54:02 EDT 2016]: Guacamole starting.
2016-07-26 14:54:03 WARN  NativeCodeLoader:62 - Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
Running on 2 inputs:
<Input #0 'synth1.normal.100k-200k.withmd' of normal dna at src/test/resources/synth1.normal.100k-200k.withmd.bam >
<Input #1 'synth1.tumor.100k-200k.withmd' of tumor dna at src/test/resources/synth1.tumor.100k-200k.withmd.bam >
Exception in thread "main" java.io.IOException: No FileSystem for scheme: file
	at org.apache.hadoop.fs.FileSystem.getFileSystemClass(FileSystem.java:2644)
	at org.apache.hadoop.fs.FileSystem.createFileSystem(FileSystem.java:2651)
	at org.apache.hadoop.fs.FileSystem.access$200(FileSystem.java:92)
	at org.apache.hadoop.fs.FileSystem$Cache.getInternal(FileSystem.java:2687)
	at org.apache.hadoop.fs.FileSystem$Cache.get(FileSystem.java:2669)
	at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:371)
	at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:170)
	at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:355)
	at org.apache.hadoop.fs.Path.getFileSystem(Path.java:295)
	at org.hammerlab.guacamole.readsets.ReadSets$.loadFromBAM(ReadSets.scala:173)
	at org.hammerlab.guacamole.readsets.ReadSets$.load(ReadSets.scala:157)
	at org.hammerlab.guacamole.readsets.ReadSets$$anonfun$4.apply(ReadSets.scala:113)
	at org.hammerlab.guacamole.readsets.ReadSets$$anonfun$4.apply(ReadSets.scala:111)
	at scala.collection.TraversableLike$WithFilter$$anonfun$map$2.apply(TraversableLike.scala:722)
	at scala.collection.Iterator$class.foreach(Iterator.scala:727)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1157)
	at scala.collection.IterableLike$class.foreach(IterableLike.scala:72)
	at scala.collection.AbstractIterable.foreach(Iterable.scala:54)
	at scala.collection.TraversableLike$WithFilter.map(TraversableLike.scala:721)
	at org.hammerlab.guacamole.readsets.ReadSets$.apply(ReadSets.scala:111)
	at org.hammerlab.guacamole.readsets.ReadSets$.apply(ReadSets.scala:96)
	at org.hammerlab.guacamole.commands.jointcaller.SomaticJoint$.inputsToReadSets(SomaticJointCaller.scala:69)
	at org.hammerlab.guacamole.commands.jointcaller.SomaticJoint$Caller$.run(SomaticJointCaller.scala:93)
	at org.hammerlab.guacamole.commands.jointcaller.SomaticJoint$Caller$.run(SomaticJointCaller.scala:77)
	at org.hammerlab.guacamole.commands.SparkCommand.run(SparkCommand.scala:12)
	at org.hammerlab.guacamole.commands.Command.run(Command.scala:46)
	at org.hammerlab.guacamole.Main$.main(Main.scala:68)
	at org.hammerlab.guacamole.Main.main(Main.scala)
       63.70 real        62.58 user         6.56 sys

**Comment by ryan-williams** [comment]
is it running for about a minute before it hits this, as the last line of your output indicates?

**Comment by timodonnell** [comment]
Yeah, that's right... I guess some sort of cache it's trying to write?

**Comment by ryan-williams** [comment]
eh no it's another shading config that I removed in #482. I'll fold the fix in to #510

**Comment by ryan-williams** [comment]
ok, new #510 should fix this too
